build/repo.targets

```diff
@@ -44,7 +44,6 @@
         <PropertyGroup>
             <_ConstantProperties>Configuration=$(Configuration)</_ConstantProperties>
             <_ConstantProperties>$(ConstantProperties);PackageVersion=$(PackageVersion)</_ConstantProperties>
-            <_ConstantProperties>$(ConstantProperties);LicenseUrl=http://example.com</_ConstantProperties>
```

Comment:
Thankfully, looking at the [satellite packages on nuget.org](https://www.nuget.org/packages/Microsoft.AspNet.SignalR.Core.zh-Hans/2.4.0), it appears that http://example.com was never used as a LicenseUrl.

Prior to 2.4.0, the satellite package linked to localized satellite licenses. That's not longer the case, but I doubt it's a big issue. I don't think anyone else has even noticed.